charts/postgres-operator/templates/postgres-operator.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
```

Comment:
directly copied from zalando could use some templating so our users can tweak the values.